buildkite/bazelci.py

```diff
@@ -3105,6 +3119,8 @@ def fetch_aggregate_incompatible_flags_test_result_command():
 
 
 def upload_project_pipeline_step(project_name, git_repository, http_config, file_config):
+    if not BAZELCI_ENABLE_REMOTE:
```

Comment:
Do we really need this change? The runner step won't invoke `upload_project_pipeline_step`.

Reply:
I wasn't able to trace the code and figure that out, I just looked for all the calls to "artifact upload" to block them.

My preference would be to separate the runner to its own python module with no buildkite dependencies, but that's a much larger refactoring on this code.

I'll remove this (and the next) and re-test to see if I get errors.